tests/functional/desktop/test_rewrites.py

```diff
@@ -2,9 +2,10 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
+
 import pytest
 import requests
-import urllib
+import urllib.parse
```

Comment:
No need to import parse. You can only import urllib and use parse by using the full path `urllib.parse`